packages/pds/src/api/app/bsky/getUserFollows.ts

```diff
@@ -19,11 +19,11 @@ export default function (server: Server) {
       let followsReq = db.db
         .selectFrom('app_bsky_follow as follow')
         .where('follow.creator', '=', creator.did)
-        .innerJoin('user_did as subject', 'subject.did', 'follow.subject')
+        .innerJoin('user_did as subject', 'subject.did', 'follow.subjectDid')
```

Comment:
You are a saint for going through all of these.

Reply:
actually this was another big win for kysely, I got compiler errors that told me exactly where these all were 😎